src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -2430,6 +1307,110 @@ export default class Aircraft {
         }
     }
 
+    /**
+     * @for AircraftInstanceModel
+     * @method aircraftTurnPhysics
+     * This turns the aircraft if it is not on the ground and has not arived at its destenation
+     */
+    aircraftTurnPhysics() {
```

Comment:
if we are updating aircraft properties, be sure to stay consistent with your naming.  This method should likely be called `updateAircraftTurnPhysics`